### ISSUE TITLE
Mangled names bubbling up to the surface

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#30]
```> (define-struct point* (x))
> (make-point* 2)
(pointƎSTAR 21)```

Do we have a way of keeping the original identifier string around somehow?

**Comment by ds26gte** [comment]
Fixed in 861f6b0.

(This feature was already in place in 82ada71 half a year ago (!), but automerge seems to have lost it without flagging a "conflict". More recent automerges are all being checked manually, but this feature was old enough to have slipped through the cracks.)

**Comment by schanzer** [comment]
Awesome. :)
